docs/scarpet/api/Scoreboard.md

```diff
@@ -21,6 +20,8 @@ scoreboard_add('lvl','level')
 
 ### `scoreboard_remove(objective)` `scoreboard_remove(objective, key)`
 
+NOTE: This function is deprecated, use `scoreboard(objective, key, null)` and `scoreboard_property(objective, 'remove')` instead.
```

Comment:
its far less intuitive to remove the objective by modifying its property, or remove entry by setting it to null.
Suggesting removing _property(..., 'remove') and just keep scoreboard_remove as is.

Reply:
alright makes sense